flink-ml-iteration/src/main/java/org/apache/flink/iteration/IterationBody.java

```diff
@@ -44,7 +51,7 @@
  * streams returned by the IterationBody.
```

Comment:
code or linkto the IterationBody?

Reply:
I change it to `iteration body`. Perhaps here we do not need the link since it is just the comment for `IterationBody`?